### ISSUE TITLE
Cross Dataset results

### ISSUE THREAD

**Issue by gufranSabri** [Visual-AI/FROSTER#17]
Im trying to reproduce the cross dataset results.

I downloaded the k400 pretrained weights from your README.
When I test on hmdb51. I get 47% top 1 accuracy instead of the 54% reported in the paper.

Following is the script i ran to get the results.
Am i missing anything?

```
ROOT=/home/g202302610/Code/FROSTER
CKPT=/home/g202302610/Code/SCAR/froster_ckpt
OUT_DIR=/home/g202302610/Code/SCAR/froster_ckpt/testing
LOAD_CKPT_FILE=/home/g202302610/Code/SCAR/froster_ckpt/froster_k400_clip_b16.pth

hmdb_file=hmdb_full
TRAIN_FILE=train.csv
VAL_FILE=val.csv
TEST_FILE=test.csv
# hmdb_file can be set as hmdb_full, hmdb_split1, hmdb_split2, hmdb_split3

cd $ROOT
python -W ignore -u tools/run_net.py \
    --cfg configs/Kinetics/TemporalCLIP_vitb16_8x16_STAdapter.yaml \
    --opts DATA.PATH_TO_DATA_DIR ./zs_label_db/hmdb_full \
    DATA.PATH_PREFIX /home/g202302610/Code/SCAR/data/hmdb51/HMDB51 \
    DATA.PATH_LABEL_SEPARATOR , \
    TRAIN_FILE $TRAIN_FILE \
    VAL_FILE $VAL_FILE \
    TEST_FILE $TEST_FILE \
    DATA.INDEX_LABEL_MAPPING_FILE /home/g202302610/Code/SCAR/label_rephrase/hmdb_rephrased_classes.json \
    TRAIN.ENABLE False \
    OUTPUT_DIR $OUT_DIR \
    TEST.BATCH_SIZE 4 \
    NUM_GPUS 1 \
    DATA.DECODING_BACKEND "pyav" \
    MODEL.NUM_CLASSES 51 \
    TEST.CUSTOM_LOAD True \
    TEST.CUSTOM_LOAD_FILE $LOAD_CKPT_FILE \
    TEST.SAVE_RESULTS_PATH temp.pyth \
    TEST.NUM_ENSEMBLE_VIEWS 3 \
    TEST.NUM_SPATIAL_CROPS 1 \
    TEST.PATCHING_MODEL False \
    TEST.CLIP_ORI_PATH /root/.cache/clip/ViT-B-16.pt \
```

**Comment by OliverHxh** [comment]
It seems weird, have you tried testing on hmdb_split1, hmdb_split2, and hmdb_split3?

**Comment by gufranSabri** [comment]
> It seems weird, have you tried testing on hmdb_split1, hmdb_split2, and hmdb_split3?

Yes i tried the splits as well. Similar decrease in score
I have a feeling there is something wrong with my dataset.

i download hmdb51 as instructed in your README. this worked fine for base to novel
but its not working for Cross Dataset. It seems some of the videos are corrupted

Can you please upload your copy of hmdb51 on google drive and send me a link. much appreciated

**Comment by OliverHxh** [comment]
I use this one to donwload [HMDB51](https://github.com/open-mmlab/mmaction2/blob/main/tools/data/hmdb51/download_videos.sh).
